scripts/screener/screener.config.js

```diff
@@ -11,9 +35,14 @@ require('tsconfig-paths').register({
   paths: compilerOptions.paths
 });
 
+// this is a fake branch to pretend we have a feature branch as a separate project
+const baseBranch = 'master-fluentui';
```

Comment:
Are you actually adding this branch? If so, how can we ensure that it stays up-to-date?

Reply:
This is a fake branch, doesn't exist in life at all. I am hoping that screener merely uses it as a key to store their baselines. They support feature branches now through this.

Reply:
This hack was an experiment, I think it isn't needed anymore.